tensorrt_llm/serve/scripts/perf_time_events/perf_time_events.py

```
@@ -0,0 +1,394 @@
+                    header = next(reader)
+                except StopIteration:
+                    continue
+                header = [h.strip() for h in header]
+                rows = [row for row in reader if row]
+        except OSError as e:
+            print(f"WARNING: failed to read KV CSV {path}: {e}")
+            continue
+
+        if base.endswith("_gen_transfer_summary.csv") or header == _GEN_SUMMARY_HEADER:
+            for row in rows:
+                rec = dict(zip(header, row))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)